packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -141,6 +145,11 @@ export class DataStores implements IDisposable {
             }
             this.contexts.addBoundOrRemoted(dataStoreContext);
         }
+        this.logger.sendTelemetryEvent({
+            eventName: "dataStoreCount",
```

Comment:
I'd give this event more generic name (ContainerLoadStats ?) such that developers could add more properties later on as needed bases.